File: 2 - Condicionais/7 - dardo.py
print('Digite as tres distancias:')

maior = 0
d1 = float(input(''))
d2 = float(input(''))
d3 = float(input(''))

if d1 > d2 and  d1 > d3:
    maior = d1
elif d2 > d1 and d2 > d3:
    maior = d2
elif d3 > d2 and d3 > d1:
    maior = d3
print(f'MAIOR DISTANCIA = {maior}')

File: C/2 - Condicionais/7 - dardo.c
#include <stdio.h>

int main() {
    
    double a, b ,c, maior;
    
    printf("Digite as tres distancias: \n");
    scanf("%lf", &a);
    scanf("%lf", &b);
    scanf("%lf", &c);

    if (a > b && a > c) {
        maior = a;
    }
    else if (b > c && b > a) {
        maior = b;
    }
    else {
        maior = c;
    }

    printf("MAIOR DISTANCIA = %.2lf", maior);

    return 0;
}